### PR TITLE
CCC: correct comments

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

**Comment by J0WI** [comment]
Some certs expired recently. Let's see if they fix it.

**Comment by numismatika** [comment]
halfnarp is still serving an expired cert.

**Comment by J0WI** [comment]
Therefore you should move this target into the comments.

**Comment by numismatika** [comment]
@J0WI all checks pass for now.

**Comment by J0WI** [comment]
This should also fix #4450

File: src/chrome/content/rules/CCC.xml [hunk 1]
```diff
@@ -10,72 +10,87 @@
...
-		- frab.camp.ccc.de
 		- hannover.ccc.de
-		- tickets.events.ccc.de
 
 -->
 <ruleset name="ccc.de (partial)">
 
 	<target host="ccc.de" />
 	<target host="events.ccc.de" />
-	<target host="rad1o.badge.events.ccc.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/CCC.xml]: WFM

File: src/chrome/content/rules/CCC.xml [hunk 2]
```diff
@@ -10,72 +10,87 @@
...
-		<!--exclusion pattern="^http://(?:desalte|ds|dsrad|wahlcomputer|wuppa)\.ccc\.de/" /-->
-
-			<!--test url="http://desalte.ccc.de/" /-->
-			<!--test url="http://ds.ccc.de/" /-->
-			<!--test url="http://dsrad.ccc.de/" /-->
-			<!--test url="http://wahlcomputer.ccc.de/" /-->
-			<!--test url="http://wuppa.ccc.de/" /-->
-
-		<!--test url="http://blog.chaosradio.ccc.de/" /-->
-		<!--test url="http://wiki.chaosradio.ccc.de/" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/CCC.xml]: Please keep a comment for it

File: src/chrome/content/rules/CCC.xml [hunk 3]
```diff
@@ -10,72 +10,87 @@
...
+	<target host="dresden.ccc.de" />
+	<target host="ftp.ccc.de" />
+	<target host="giessen.ccc.de" />
+	<target host="eh2003.hamburg.ccc.de" />
+	<target host="hamburg.ccc.de" />
+	<target host="www.hamburg.ccc.de" />
+	<target host="hannover.ccc.de" />
+	<target host="karlsruhe.ccc.de" />
+	<target host="kassel.ccc.de" />
+	<target host="koeln.ccc.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/CCC.xml]: api.koeln.ccc.de

File: src/chrome/content/rules/CCC.xml [hunk 4]
```diff
@@ -10,72 +10,87 @@
...
-		- berlin.ftp.media ²
 		- wuppa ¹
+		- irc.darmstadt
+		- www.duesseldorf ²
+		- frankfurt ² 
+		- ftp.media ² 
+		 -upload.media ² 
+		- gost.hannover ²
+		- www.hannover ²
+		- pads.ccc.de ³
```
Comment (J0WI) [inline src/chrome/content/rules/CCC.xml]: please remove `ccc.de` for consistency (arr add it everywhere).

**Comment by J0WI** [comment]
Please also remove the obsolete `CCC.de-cacert.xml`.

**Comment by numismatika** [comment]
@J0WI Cacert file is deleted

**Pull request merged**